docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM php:8.3-fpm
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip \
+    libicu-dev \
+    libzip-dev \
+    nano \
+    libjpeg62-turbo-dev \
+    xdg-utils \
+    supervisor  # Adding supervisor
+
+# Configure & install gd extension with jpeg support
+RUN docker-php-ext-configure gd --with-jpeg \
+    && docker-php-ext-install gd
+
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath zip \
+    && docker-php-ext-configure intl \
+    && docker-php-ext-install intl
+
+# Clear installation sources to reduce image size
+RUN docker-php-source delete
+
+# Install Xdebug and configure
+RUN pecl install xdebug \
+    && docker-php-ext-enable xdebug
+
+
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Get latest Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin/ --filename=composer
```

Comment:
Maybe you can copy it from the official composer docker: 
`COPY --from=composer:2.5.8 /usr/bin/composer /usr/bin/composer`

Reply:
I ti is the same I use curl call to copy it from official page

Reply:
What Bobi suggested is the new way of doing the same thing as you did but it's much smaller in package size which may lower the build time slightly.

Reply:
code update